### ISSUE TITLE
Filters not filtering as expected

### ISSUE THREAD

**Issue by jappyjan** [appwrite/appwrite#1325]
## 🐛 Bug Report

I have a collection in which i store latitude and longitude vales.
I filter those with boundaries.
As soon as i filter by all 4 required boundaries, i get 0 results back even though i should get one back.

## To Reproduce

Add a Collection with properties lat and lng (numeric)
Add at least one item to the collection with the following values

lat: 53.5817
lng: 9.9887

Query database with the following code

´´´
const filters = [
   "lat>=53.572712232909396",
   "lat<=53.590685861237205",
   "lng>=9.973414583625384",
   "lng<=10.003985416373155",
];

console.log('filters', filters);

const response = await appwrite.database.listDocuments(
  <id of my collection>,
  filters
);

const tags = response.documents;

console.log('fetched tags', tags);
´´´

## Expected behavior

i expect to get back the items of my collections as the filters match their properties.

## Actual Behavior

i only get back my items, if i remove the fourth filter.
otherwise i get back an empty array.

## Your Environment

hosted at DigitalOcean via the auto deploy thingy.
environment variables all standard except for smtp ones.
should be all up to date as it was installed this morning.

**Comment by TorstenDittmann** [comment]
Hello @jappyjan 👋🏻 

Do you mind sharing your collection rules JSON?

This way we can re-create your environment 🙇🏻 

Greetings Torsten

**Comment by jappyjan** [comment]
Here ya go

```
{
    "$id": "60d783e826e77",
    "$permissions": {
        "read": [
            "*"
        ],
        "write": [
            "*"
        ]
    },
    "name": "Tags to Review",
    "dateCreated": 1624736744,
    "dateUpdated": 1624736981,
    "rules": [
        {
            "$id": "60d7847e74e80",
            "$collection": "rules",
            "type": "numeric",
            "key": "lat",
            "label": "Latitude",
            "default": 0,
            "array": false,
            "required": true,
            "list": []
        },
        {
            "$id": "60d7847e7652f",
            "$collection": "rules",
            "type": "numeric",
            "key": "lng",
            "label": "Longitude",
            "default": 0,
            "array": false,
            "required": true,
            "list": []
        },
        {
            "$id": "60d7847e78853",
            "$collection": "rules",
            "type": "text",
            "key": "title",
            "label": "Title",
            "default": "",
            "array": false,
            "required": false,
            "list": []
        },
        {
            "$id": "60d7847e7a1a5",
            "$collection": "rules",
            "type": "text",
            "key": "pictures",
            "label": "Pictures",
            "default": "",
            "array": true,
            "required": true,
            "list": []
        }
    ]
}
```

**Comment by TorstenDittmann** [comment]
Looks like this filter alone `lng<=10.003985416373155` will return 0 documents 🤔 

Investigating it more

**Comment by TorstenDittmann** [comment]
> Looks like this filter alone `lng<=10.003985416373155` will return 0 documents 🤔
> 
> Investigating it more

Looks like a bug! Using `lng<=9.9934145836253841` will work without a problem.

**Comment by eldadfux** [comment]
This is now fixed and available as part of Appwrite 0.9:
https://dev.to/appwrite/announcing-appwrite-0-9-the-open-source-firebase-alternative-53ho 🥳 🎉
